docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md

```diff
@@ -148,8 +152,8 @@ You can use your own custom VPCs. This allows you the highest level of customiza
 
 - A VPC ID to use for each region.
 - A Security Group ID to use for each region. This is attached to all YugabyteDB nodes and must allow traffic from all other YugabyteDB nodes, even across regions, if you deploy across multiple regions.
-- A mapping of what Subnet IDs to use for each Availability Zone in which you wish to be able to deploy. This is required to ensure that YugabyteDB Anywhere can deploy nodes in the correct network isolation that you desire in your environment.
 - A custom AMI ID to use in each region. For a non-exhaustive list of options, see [Ubuntu 18 and Oracle Linux 8 support](#ubuntu-18-and-oracle-linux-8-support). If you do not provide any values, a recent [AWS Marketplace CentOS AMI](https://wiki.centos.org/Cloud/AWS) is used.
```

Comment:
TODO: 
Ah sorry, I should've added more context for this in our doc - I'll need to add some more information here:

We used to always provide custom AMI ID as an optional field.
Now, `Custom AMI ID` is only collected when the user selects `Custom AMI` in the radio button group (other options are `Default AArch64 AMI` and `Default x86_64 AMI`).

This is related to the backend work for adding platform support for ARM done by @chidmuthu